misc/multihash/src/hashes.rs

```diff
@@ -37,8 +37,8 @@ pub enum Hash {
 
 impl Hash {
     /// Get the corresponding hash code.
-    pub fn code(&self) -> u16 {
-        match *self {
+    pub fn code(self) -> u16 {
```

Comment:
I think this is the only breaking change and it would force us to do a "major" bump, so it feels a bit stupid.